canvas_modules/common-canvas/__tests__/test_resources/paramDefs/custom-ctrl-op_paramDef.json

```diff
@@ -83,6 +98,25 @@
           "type": "string"
         }
       ]
+    },
+    {
```

Comment:
Is there any way to change the values in this table?  If not, do we need it?

Reply:
In DataStage, this is a readonlytable. They edit the values in `Case Sensitive` column in a tearsheet. But in this PR, Case sensitive column is editable. 

Code works on cell level though. After updating second row, I expected entire column to be hidden but it hides only 2nd row cell - 
![hide-col-cell-level](https://user-images.githubusercontent.com/25124000/172721490-60eb371d-9774-4548-a36d-71f5994c80d2.gif)

Reply:
Do we really need this custom control/operator?  Can't this just be done with a checkbox that hides the case sensitive column?

Reply:
That worked! Removed custom control and added a checkbox in `readonlyTable_paramDef.json` to hide table column - 
![Screen Shot 2022-06-09 at 12 22 00 PM](https://user-images.githubusercontent.com/25124000/172927858-750997a9-af01-4507-ad89-8f014c9e5112.png)